### PR TITLE
Revert "Started initial imp of test cae 001"

### PR DESCRIPTION
**Pull request by aysen0513**
Hi guys

### CODE REVIEW

File: src/test/java/TC_Aysen.java [hunk 1]
```diff
@@ -1,5 +0,0 @@
-public class TC_Aysen {
-    public static void main(String[] args) {
-        System.out.println("Working on");
```
Comment (sbabayev) [inline src/test/java/TC_Aysen.java:3]: this is sample review

**Review by sbabayev — APPROVED** [review]

**Comment by aysen0513** [comment]
We're awesome...

File: src/test/java/TC_Aysen.java (hunk 1 above)
Comment (aysen0513) [reply src/test/java/TC_Aysen.java:3]: Yes you're right.

**Comment by aysen0513** [comment]
How is it going?

**Review by aysen0513 — COMMENTED** [review]
Hi Group_22

**Pull request merged**